CHANGELOG.md

```diff
@@ -269,3 +270,34 @@ The changes of each release is tracked by this file.
  - [[```43d62f06```](https://github.com/jina-ai/finetuner/commit/43d62f068fdf80ca0a1a4d9f86ec24804f7f6aca)] __-__ __readme__: update logo (*Han Xiao*)
  - [[```489014ee```](https://github.com/jina-ai/finetuner/commit/489014ee4e12e6c3cc697d7c4da4129dd600ccdb)] __-__ __version__: the next version will be 0.1.4 (*Jina Dev Bot*)
 
+<a name=release-note-0-1-5></a>
```

Comment:
?

Reply:
no idea, i think it's somethign related to jina bot

Reply:
I see - I think this came when you merged with my branch for some reason. It's strange because this is alreday part of my branch